samples/guice/src/main/webapp/login.jsp

```diff
@@ -58,11 +61,12 @@
     </style>
 
 
-    <table class="sample">
+    <table class="sample" style="padding: 10px;">
+    	<caption style="padding: 10px;"> Sample accounts </caption>
         <thead>
         <tr>
-            <th>Username</th>
-            <th>Password</th>
+            <th id="username">Username</th>
+            <th id="password">Password</th>
```

Comment:
Can you explain this change?  I'm guessing if `id` attributes are set they should probably be set on the actual inputs?

I don't have the context of why this change is suggested, by my assumption adding the `id` attr base it easier to drive test frameworks like Selenium (so a `getById()` can be used instead of a more complicated selector?)